client/src/components/Panels/utilities.js

```diff
@@ -1,86 +1,136 @@
-export function filterToolSections(layout, results) {
-    if (results) {
-        results = normalize_results(results);
-        const filteredLayout = layout.map((section) => {
-            var toolRes = [];
-            if (section.elems) {
-                section.elems.forEach((el) => {
-                    if (
-                        (!el.text && results.includes(el.id)) ||
-                        (el.tool_shed_repository && results.includes(el.tool_shed_repository.name))
```

Comment:
Just a note to double-check (with a test, ideally) the logic showing favorites independent of version; xref https://github.com/galaxyproject/galaxy/pull/9873

Reply:
@dannon Confirmed. The tools filter methods will still work with the Favorites independent of version. The front-end unit test attached to this PR includes coverage for matching the Tools store to the Favorites results (as of course it includes coverage for matching the Tools store to the Whoosh Search results).